### ISSUE TITLE
Problem with windows background

### ISSUE THREAD

**Issue by entuccio** [NtsFranz/LoneEchoSubtitles#3]
Hello,  after I installed Spark, and pinned the window, also did "streamer mode" how can I remove the green/dark background behind the subtitles?  The window disturbs me while I'm playing. Thank you for your creation.

**Comment by NtsFranz** [comment]
Unfortunately Oculus Dash has no way to make windows have a transparent background.
The green background is for people using OBS to add a greenscreen filter.
Your best option is just to use the black background in-game.

**Comment by entuccio** [comment]
Right, but What about [this video](https://www.youtube.com/watch?v=4UVqUG6hlWc&t=162s) ?

What do you mean with background in-game?

**Comment by NtsFranz** [comment]
That video is overlaying the subtitles using OBS. They are not visible to me in VR.
I just mean that if you're using the subtitles window in game, the black background is less distracting than the green one.

**Comment by entuccio** [comment]
Thank you very much.
